samplesheets/tests/test_permissions_api.py

```diff
@@ -25,7 +28,10 @@
     REMOTE_SITE_SECRET,
     INVALID_SECRET,
 )
-
+from samplesheets.tests.test_permissions_api_taskflow import (
+    IrodsAccessTicketAPIViewTestBase,
```

Comment:
Good work moving the tests, but we should **not** import the base class from taskflow tests. That will just set up the test in iRODS again, which is exactly what we wanted to avoid :)

Instead, use the standard base class for non-taskflow view tests, or create your own override if that's needed.